The java 11 test failure is unrelated:

```
kafka.admin.ResetConsumerGroupOffsetTest.testResetOffsetsToEarliestOnOneTopicAndPartition

Error Message
java.lang.AssertionError: expected:<Map(bar-0 -> 50, bar-1 -> 0)> but was:<Map(bar-0 -> 50, bar-1 -> 50)>
Stacktrace
java.lang.AssertionError: expected:<Map(bar-0 -> 50, bar-1 -> 0)> but was:<Map(bar-0 -> 50, bar-1 -> 50)>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:118)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at kafka.admin.ResetConsumerGroupOffsetTest.resetAndAssertOffsetsCommitted(ResetConsumerGroupOffsetTest.scala:388)
	at kafka.admin.ResetConsumerGroupOffsetTest.testResetOffsetsToEarliestOnOneTopicAndPartition(ResetConsumerGroupOffsetTest.scala:247)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
```

https://builds.apache.org/job/kafka-pr-jdk11-scala2.12/1162/

--- a/streams/src/main/java/org/apache/kafka/streams/kstream/internals/TupleForwarder.java
+++ b/streams/src/main/java/org/apache/kafka/streams/kstream/internals/TupleForwarder.java
@@ -57,8 +57,9 @@ class TupleForwarder<K, V> {
                 wrapped = ((WrappedStateStore) wrapped).wrappedStore();
             }
 
-            if (!(wrapped instanceof CachedStateStore))
+            if (!(wrapped instanceof CachedStateStore)) {
                 return null;
+            }
 
             return (CachedStateStore) wrapped;
         }

